dse-k8s-operator/README.md

```diff
@@ -287,82 +317,102 @@ spec:
 with the default username and password in place is extremely dangerous. Its
-strongly recommended to protect your DSE cluster with a network firewall during
+strongly recommended to protect your cluster with a network firewall during
 deployment, and [secure the default superuser
 account](https://docs.datastax.com/en/security/6.7/security/Auth/secCreateRootAccount.html)
 before exposing any ports publicly.
 
 ## Scale up
 
-The `size` parameter on the `DseDatacenter` determines how many DSE instances
-are present in the datacenter. To add more DSE nodes, edit the YAML file from
+The `size` parameter on the `CassandraDatacenter` determines how many server nodes
+are present in the datacenter. To add more nodes, edit the YAML file from
 the `Example Config` section above, and re-apply it precisely as before. The
-operator will add DSE pods to your datacenter, provided there are sufficient
+operator will add pods to your datacenter, provided there are sufficient
 Kubernetes worker nodes available.
 
-For racks to act effectively as a fault-containment zone, each rack in the DSE
-cluster must contain the same number of DSE instances.
+For racks to act effectively as a fault-containment zone, each rack in the
+cluster must contain the same number of instances.
 
-## Change DSE configuration
+## Change server configuration
 
-To change the DSE configuration, update the `DseDatacenter` and edit the
+To change the database configuration, update the `CassandraDatacenter` and edit the
 `config` section of the `spec`. The operator will update the config and restart
 one node at a time in a rolling fashion.
 
 ## Multiple Datacenters in one Cluster
 
-To make a multi-datacenter cluster, create two `DseDatacenter` resources and
-give them the same `dseClusterName` in the `spec`.
+To make a multi-datacenter cluster, create two `CassandraDatacenter` resources and
+give them the same `clusterName` in the `spec`.
 
 _Note that multi-region clusters and advanced workloads are not supported, which
-makes many multi-dc use-cases inappropriate for the operator._
+makes many multi-DC use-cases inappropriate for the operator._
 
-# Maintaining Your DSE Cluster
+# Maintaining Your Cluster
 
-## Repair
+## Data Repair
 
 The operator does not automate the process of performing traditional repairs on
-keyspace ranges where the data has become inconsistent due to a DSE instance
+keyspace ranges where the data has become inconsistent due to an instance
 becoming unavailable in the past.
 
-Instead, DSE provides
+DSE provides
 [NodeSync](https://www.datastax.com/2018/04/dse-nodesync-operational-simplicity-at-its-best),
 a continuous background repair service that is declarative and
 self-orchestrating. After creating your cluster, [Enable
 NodeSync](https://docs.datastax.com/en/dse/6.7/dse-admin/datastax_enterprise/tools/dseNodesync/dseNodesyncEnable.html)
 on all new tables.
 
+In the future the operator will support deploying http://cassandra-reaper.io/ for
```

Comment:
This should be removed as it discusses roadmap. I would include something to the effect of:

>Future releases may include integration with open source repair services.